experimental/dds/tree2/src/feature-libraries/editable-tree-2/lazyField.ts

```diff
@@ -228,6 +228,18 @@ export abstract class LazyField<TKind extends FieldKindTypes, TTypes extends All
 	public getFieldPath(): FieldUpPath {
 		return this[cursorSymbol].getFieldPath();
 	}
+
+	/**
+	 * Throws iff the field is under a node whose tree status is not `TreeStatus.InDocument`.
+	 * Does not throw for the field that contains the document root.
```

Comment:
```suggestion
	 ```suggestion
	 * Returns the path to this field to use for editing. Throws iff this path is not {@link TreeStatus#InDocument}.
	 * This path is not valid to hold onto across edits: this must be recalled for each edit.
```
```